CMake/kwiver-config-build.cmake.in

```diff
@@ -14,11 +14,8 @@ set(CMAKE_MODULE_PATH "@KWIVER_CMAKE_DIR@" "${CMAKE_MODULE_PATH}")
 
 # KWIVER include directory
 set(KWIVER_INCLUDE_DIRS
-  "@KWIVER_SOURCE_DIR@" "@KWIVER_SOURCE_DIR@/vital"
-  "@KWIVER_BINARY_DIR@" "@KWIVER_BINARY_DIR@/vital"
-  "@KWIVER_BINARY_DIR@/vital/kwiversys"
-  "@KWIVER_SOURCE_DIR@/sprokit/src"
-  "@KWIVER_BINARY_DIR@/sprokit/src"
```

Comment:
With this include path, sprokit includes still (must) start with sprokit/... since the path is .../sprokit/src/sprokit/...
But maybe this is not a long term problem since the eventual plan is to refactor the sprokit directory to get "sprokit/src" out of the path.